pkg/ipmi/ipmi.go

```diff
@@ -200,35 +204,50 @@ func fdSet(fd uintptr, p *syscall.FdSet) {
 }
 
 func (i *IPMI) sendrecv(req *req) ([]byte, error) {
```

Comment:
I haven't been able to test if these changes affect the rest of the functions in this file, is there a good way for me to do that?

Reply:
cc @pmazzini @JingleHsuWiwynn